stravalib/client.py

```diff
@@ -154,16 +160,15 @@ def exchange_code_for_token(self, client_id, client_secret, code):
 
         Returns
         -------
-        Dictionary containing the access_token, refresh_token
-        and expires_at (number of seconds since Epoch when the provided access token will expire)
+
+
```

Comment:
```suggestion
        Dictionary containing the access_token, refresh_token
        and expires_at (number of seconds since Epoch when the provided access token will expire)
```